test/utils.py

```diff
@@ -16,7 +16,9 @@
 ###
 
 import yaml
-from mock import mock, patch
+import mock
+
+from mock import patch
```

Comment:
👍  I`ll merge this code then we add another PR with code refactoring for this class